app/models/manageiq/providers/vmware/infra_manager/provision_workflow/dialog_field_validation.rb

```diff
@@ -0,0 +1,15 @@
+# These methods are available for dialog field validation, do not erase.
+module ManageIQ::Providers::Vmware::InfraManager::ProvisionWorkflow::DialogFieldValidation
+  def validate_placement_host_name(field, values, dlg, fld, value)
+    result = validate_placement(field, values, dlg, fld, value)
+    return result if result.nil?
+
+    ems_cluster = EmsCluster.find_by(:id => get_value(values[:placement_cluster_name]))
+
+    if ems_cluster.nil?
+      _("Either Host Name or Cluster Name is required")
+    elsif !ems_cluster.drs_enabled
+      _("%{field_required} required for Non-DRS enabled cluster") % {:field_required => result}
```

Comment:
`field_required` should render out to something like `Environment/Host is required`, so you should also drop the "required" in your string otherwise you'll get `Environment/Host is required required for Non-DRS enabled cluster`